### DISCUSSION TITLE
When I run it, it shows me an error due to Java version conflict.

### DISCUSSION THREAD

**Discussion by danieladariz1** [mobile-dev-inc/Maestro#294] [Q&A]
when i execute command : maestro test login.yaml
 Response: 
Exception in thread "main" java.lang.UnsupportedClassVersionError: maestro_android/MaestroDriverGrpc has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

SO: MACBook Air

Can you help me with this problem?

**Comment by kaling852** [comment]
I ran into the same problem with my non-m1 mac. 
It is pretty much saying you have java 1.8 and you need java 11 or above to make this work. ([version reference](https://docs.oracle.com/javase/specs/jvms/se16/html/jvms-4.html#jvms-4.1-200-B.2))

You need to download the Java 11 or above and then make sure java_home is pointing to the newer version and not the 1.8 
you can check the "Home" version by running `java -version` in terminal 

to se the JAVA_HOME,  this stackoverflow link should help  https://stackoverflow.com/questions/32517507/how-to-set-java-home-environment-variable-on-macos

**Comment by ArthurSav** [comment]
Hi, I believe the current min JAVA version has been downgraded to 8. So it should be possible to run with either 8 or 11
